util/builtinobjects/builtinobjects.go

```diff
@@ -243,7 +244,7 @@ func (b *builtinObjects) inject(ctx context.Context, spaceID string, useCase pb.
 		}
 	}()
 
-	if err = b.importArchive(ctx, spaceID, path); err != nil {
+	if err = b.importArchive(context.Background(), spaceID, path); err != nil {
```

Comment:
Is using other context.Context from what comes in ObejctImportUseCase safe here?

Reply:
I think it's ok, did you test it?